mainnet/index.js

```diff
@@ -356,25 +411,29 @@ const walletApiMethods = {
 
 const workerApiMethods = {
   AddPiece: {},
-  Closing: {
```

Comment:
Any idea why this has changed? A removal is suspicious.

Reply:
This was removed in https://github.com/filecoin-project/lotus/pull/3618/files#diff-01f92d1a4e8802e8d7b9c8e97e5d9e1b3b5b28b12eb1a2edbc9e6525bec9becbL39

Probably because the Worker changed to be async